app/javascript/src/components/TutorialPage.tsx

```diff
@@ -55,30 +56,49 @@ const pages: TutorialPageContent[] = [
   {
     title: '#5 - Advance Choice Logic',
     text: [
-      'You can attach modifiers to choices, and use those modifiers to hide and show choices in other scenes.',
-      'For example, the “Teleport to Moon” choice might only be available to the reader if the “has teleporter” modifier was picked up in an earlier choice.'
+      'When it comes to choices, you have the option to use some advanced logic.',
+      'For example, you can make a choice available only if the player has a particular item.',
+      'You can also add stats to your player depending on their choices.'
     ],
-    image: require('../images/tutorial/advanced.gif')
+    image: require('../images/tutorial/introduction_to_advanced_logic.gif')
   },
   {
-    title: '#6 - Saving',
+    title: '#6 - Making Use of Items',
+    text: [
+      'You can directly configure a choice to add or remove an item.',
+      'This allows you to add conditional logic based on whether the item is available.'
+    ],
+    image: require('../images/tutorial/using_items.gif'),
+    url: 'https://storyboard.viget.com/items-example'
+  },
+  {
+    title: '#7 - How Stats Work',
+    text: [
+      'Stats give you the option to create a more interactive experience.',
+      'For example, you are presented with two shields and only one of them will give you enough \'Defense\' to protect you.',
+    ],
+    image: require('../images/tutorial/using_stats.gif'),
+    url: 'https://storyboard.viget.com/stats-example'
+  },
```

Comment:
I think the list could use a bit of a shuffle here, think you could rig up this order? 
```
'#1 - Scenes'
'#2 - Choices and Links'
'#3 - Moving and Deleting Things'
'#4 - Navigation'
'#5 - Editing Story Details'
'#9 - Saving'
'#6 - Advanced - Using Items'
'#7 - Advanced - Using Stats'
'#8 - Advanced - Using Conditions'
'#10 - Time to Play!'
'#11 - Disclaimer'
```

Reply:
I just noticed the numbers are a bit off in that list but the titles make sense (in terms of order). Should I go down the list according to the titles?

Reply:
I went ahead and added a specific 'Advanced - Using Conditions' page since that would be useful. I ended up removing the intro to advanced logic page and ordered them according to the titles (since those made sense). 

Let me know if you want me to change it around!